app/models/kljb/person.rb

```diff
@@ -11,4 +11,8 @@ module Kljb::Person
   included do
     Person::PUBLIC_ATTRS -= [:nickname]
   end
+
+  def underage?
```

Comment:
Machts Sinn das cutoff date hier als Param mit default value zu definieren?
Ohne Vorwissen würde ich eigentlich immer davon ausgehen, dass es sich auf das aktuelle Datum bezieht und nicht auf ein "magic date" in einer Domain Klasse...

Reply:
Für mich war es praktisch, weil sich dann der Code zum Filtern nach Volljährigkeit zum Stichtag besser liest. Wenn wir das mit Parameter machen und damit das "magic-date" mehr in den Fokus rücken, würde ich den ganzen Methodenbody von  `underage?` in den Block verschieben, der die Filterung macht. Dann ist dort auch klar, was genau die Bedingung ist. 

Ich gehe nicht davon aus, dass wir `underage?` noch an vielen anderen Stellen brauchen, daher: go ahead.

Reply:
Das `select` hier: https://github.com/hitobito/hitobito_kljb/pull/14/files#diff-58c0bc5d499a529ea98458b962937bf9360ef8c950729f5afb89c11aeae57637R67 würde dann einen echten Block bekommen.